doc/api/documentation.md

```diff
@@ -57,12 +57,8 @@ attaching a listener to the [`'warning'`][] event.
 added: v0.6.12
 -->
 
-> Stability: 1 - Experimental
-
-Every `.html` document has a corresponding `.json` document presenting
-the same information in a structured manner. This feature is
-experimental, and added for the benefit of IDEs and other utilities that
-wish to do programmatic things with the documentation.
+Every `.html` document has a corresponding `.json` document. This is for IDEs
+and other utilities doing programmatic things with the documentation.
```

Comment:
While you're here, could we get rid of "doing programmatic things?" How about "consuming the documentation programmatically?"

Reply:
Sure! I went a step further and went with "that consume the documentation". Hope I'm not GOING TOO FAR!!!!